### PR TITLE
remove default export

### PR DESCRIPTION
**Pull request by GregTheGreek**
Removes `hash` as the default export and add web support.

### CODE REVIEW

File: src/wasm.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
-const fs = require("fs");
-const loader = require("assemblyscript/lib/loader");
-const compiled = new WebAssembly.Module(fs.readFileSync(__dirname + "/../build/optimized.wasm"));
-const imports = {};
-Object.defineProperty(module, "exports", {
-    get: () => loader.instantiate(compiled, imports)
-});
+try {
+  module.exports = require("./web");
```
Comment (MaxGraey) [inline src/wasm.js]: just use
```ts
module.exports = typeof window !== 'undefined' ? require("./web") : require("./node");
```
Or
```ts
module.exports = typeof process !== 'undefined' ? require("./node") : require("./web");
```

Comment (GregTheGreek) [reply src/wasm.js]: good call!

**Comment by tueric** [comment]
Are you sure the browser version work, @GregTheGreek ? I can't seem to get it working...

**Comment by GregTheGreek** [comment]
@tueric debugging..

**Pull request closed without merging**